li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/producer/LiKafkaFederatedProducerImpl.java

```diff
@@ -111,6 +119,12 @@ public boolean closed() {
   // Per cluster producers, which sends raw bytes
   private Map<ClusterDescriptor, PerClusterProducerContext<K, V>> _producers;
 
+  // Raw byte producer builder for creating per-cluster producer
+  private RawByteProducerBuilder _producerBuilder;
```

Comment:
nit - just "RawProducerBuilder" should be clear enough, and is shorter

Reply:
Done